docs/administration/managing-users-and-teams/creating-teams-for-a-user-with-mixed-environment-privileges.md

```diff
@@ -4,39 +4,47 @@ description: This guide describes how to create a teams for a user with mixed en
+![](add-team-detail.png)
 
-With the permission rules configured add the users that you want them to apply to. Remember that if the user is a member of another team that has the same roles but for the production environment, then the permissions will be effective for both teams. The team permissions are all additive.
+Give the team an appropriate name like *Developers* and click **Save**. 
 
-![](/docs/images/guides-user-role-and-teams-configuration/dev-deployers-members.png "width=500")
+## Adding the Project Viewer role for all environments 
 
-When you are happy with these changes hit **Save** to make them effective.
+We can now add the **Project viewer** role to 
+all environments by clicking **Include user role** from the **User Roles** tab. This role provides read only access to deployment
+processes and releases. Because we will not provide any scoping for this role - this role will form the baseline permissions 
+for this team in any scope. 
 
-## Creating the "*Prod Deployment Viewers*" Team {#Creatingteamsforauserwithmixedenvironmentprivileges-CreatingThe&quot;ProdDeploymentViewers&quot;Team}
+![](add-unscoped-role.png)
 
-The previous rule was all well and good, but what about if we now want to allow the developer to see what has been deployed to production while still preventing them from pushing releases out to that stage. We can't go and modify the *Dev Deployers* team because that contains rules scoped to specific environments. Instead we will create a new team that is scoped to production, but only contains view permissions.
+## Adding additional roles for a subset of environments
+Since our goal is to give members of the Developers team the ability to create & deploy releases _in the Development and 
+Staging environments only_, we can click **Include user role** again, this time adding the **Project deployer** role. 
+This role provides all the permissions of the **Project viewer** role as well as allowing a team member to deploy a release. 
+This time, we will click on **Define Scope** and choose the environments that we would like to scope the role to, before 
+hitting the **Apply** button.
+![](define-scope-for-user-role.png)
 
-Once again go back to the **Teams** tab and click **Add team**. Lets give it a name that describes its purpose, *Prod Deployment Viewers*, and this time add just the **Project viewer** role. This allows the user to see details of a project, but not actually make any modifications.
+We can repeat this process for the **Project lead** role to also give this team release creation permissions in the Development 
```

Comment:
This example seems a bit weird (admittedly it was weird before too). Since Project Lead extends Project Deployer, it was redundant to add Project Deployer in the first place. I think we can and should just get rid of the Project Lead part of this example entirely.

Reply:
Good point! Will make it so.